styles/style.css

```diff
@@ -0,0 +1,66 @@
+/* style.css */
+
+.first-container {
+    max-height: 700px;
+    
+    border: lightblue 2px solid;
+    display: grid;
+    grid-template: 100% / 50%, 50%;
+}
+
+.first-container ul {
+    grid-column-start: 2;
+    line-height: 2;   
+}
+
+.first-container p {
+    max-height: 500px;
+    font-size: xx-large;
+    font-style: italic;
+    color: blue;
+    line-height: 3;
+    
+}
+
+.second-container {
+    min-height: 300px;
+    width: 700px; 
+    line-height: 3;
+}
+
+.flex-container {
+    display: flex;
+    flex-direction: column; 
+}
+
+.flex-container-row {
+    display: flex;
+    justify-content: space-between;
+    width: 450px;
+}
+
+body {
+    font-family: sans-serif;
+    background-color: rgb(248, 250, 238);
+}
+footer {
+    background-color:rgb(196, 255, 207);
+    border: 1px honeydew    
+}
+
+h1 {
+    font-size: rem;
```

Comment:
we must give the font-size a value, not just measurement unit. Something like `1rem` perhaps